src/main/eo/org/eolang/txt/text.eo

```diff
@@ -38,48 +38,56 @@
       ^.s.slice start len
 
   # Trim it from both sides
-  # @todo #97:30min Current implementation trims only left part of spaces in the given string.
-  #  We should increase realization and trim right part of spaces also.
-  #  After than we should unwrap disabled trim tests
+  # @todo #107:30min Current implementation of trimmed method catches stackoverflow
+  #  for tests where both sides of strings are changing or just more than one space
+  #  should be removed. After fixing it unwrap nop from all trimmed tests
   [] > trimmed
-    reducedi. > spaces!
-      list
-        bytes-as-array
-          as-bytes.
-            s
-      *
-      [a i x]
-        up-cased.ascii > code!
-          x
-        32 > space-code!
-        if. > @
-          eq.
-            code
-            space-code
-          with.
-            a
-            i
-          a
-    reduced. > left-pos!
-      list
-        spaces
-      0
-      [a x]
-        if. > @
-          eq.
-            a
-            x
-          plus.
-            a
-            1
-          a
-    slice. > @
+    rec-trim > @
       s
-      left-pos
-      minus.
+
+    [str] > rec-trim
```

Comment:
@EugeneDar Maybe we can create constant from `str` object and use it everywhere in the body of the recursive function?

Reply:
@Graur since each ticket is designed for half an hour, it is probably worth leaving this solution to an already created puzzle.

Reply:
@EugeneDar I meant to create `str > const!` inside `rec-trim` object and to use `const` instead of `str` there. To avoid multiply dataization of `str` object inside the `rec-trim` object.

Reply:
@Graur we still have problems with this tests. I will add a puzzle about it.

Reply:
@Graur sorry, we actually have such a puzzle, so I think we can close current ticket, aren't we?